lib/ble.toit

```diff
@@ -257,23 +270,157 @@ class Client:
     state := resource_state_.wait_for_state STARTED_EVENT_
     resource_state_.clear_state STARTED_EVENT_
 
+class ServerConfig:
+  resource_group_ := null
+  services/List := []
+
+  constructor:
+    resource_group_ = ble_server_config_init_
```

Comment:
Should this be disposed when ServerConfig goes away? It looks like a leak because the resource group will not go away before the current process dies, unless we add code to clean it up. It looks like we will need to keep it around as long as there are registered services, so maybe we should let the Service object hold a pointer to the ServerConfig it came from instead of just holding on to the resource_group. That way, we can add finalization behaviour to ServerConfig that cleans this up.